app/assets/stylesheets/blacklight/hierarchy/hierarchy.scss

```diff
@@ -34,8 +34,21 @@ $text-muted: #777 !default;
     }
   }
 
+  .twiddle>.toggle-handle {
+    background-color: transparent;
+  }
+
+  .twiddle>.toggle-handle .toggle-icon {
+    background-position: center;
+    background-repeat: no-repeat;
+    margin-top: 3px;
+    min-height: 20px;
+    min-width: 20px;
+  }
+
   .twiddle>.toggle-handle .closed {
-    display: inline;
+    background-image: url('data:image/svg+xml, <svg aria-hidden="true" focusable="false" data-prefix="far" data-icon="plus-square" class="svg-inline--fa fa-plus-square fa-w-14" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 448 512"><path fill="currentColor" d="M352 240v32c0 6.6-5.4 12-12 12h-88v88c0 6.6-5.4 12-12 12h-32c-6.6 0-12-5.4-12-12v-88h-88c-6.6 0-12-5.4-12-12v-32c0-6.6 5.4-12 12-12h88v-88c0-6.6 5.4-12 12-12h32c6.6 0 12 5.4 12 12v88h88c6.6 0 12 5.4 12 12zm96-160v352c0 26.5-21.5 48-48 48H48c-26.5 0-48-21.5-48-48V80c0-26.5 21.5-48 48-48h352c26.5 0 48 21.5 48 48zm-48 346V86c0-3.3-2.7-6-6-6H54c-3.3 0-6 2.7-6 6v340c0 3.3 2.7 6 6 6h340c3.3 0 6-2.7 6-6z"></path></svg>');
```

Comment:
Would it be smart to put this in a sass variable?

Reply:
👍 ; that'd make it easier for downstream consumers to configure too.

Reply:
I think it'd also be good and appropriate to retain the fontawesome attribution in the SVG content itself. The license on https://github.com/FortAwesome/Font-Awesome#license suggests that's sufficient attribution 🤷‍♂️